src/network/networkctl.c

```diff
@@ -2638,24 +2667,31 @@ static int link_lldp_status(int argc, char *argv[], void *userdata) {
                 return c;
 
         pager_open(arg_pager_flags);
+        if (arg_json_format_flags == JSON_FORMAT_OFF) {
+                TableCell *cell;
+
+                udata.table = table_new("link",
+                                        "chassis-id",
+                                        "system-name",
+                                        "caps",
+                                        "port-id",
+                                        "port-description");
+                if (!udata.table)
+                        return log_oom();
 
-        udata.table = table_new("link",
-                                "chassis-id",
-                                "system-name",
-                                "caps",
-                                "port-id",
-                                "port-description");
-        if (!udata.table)
-                return log_oom();
+                if (arg_full)
+                        table_set_width(udata.table, 0);
 
-        if (arg_full)
-                table_set_width(udata.table, 0);
+                table_set_header(udata.table, arg_legend);
 
-        table_set_header(udata.table, arg_legend);
-
-        assert_se(cell = table_get_cell(udata.table, 0, 3));
-        table_set_minimum_width(udata.table, cell, 11);
-        table_set_ersatz_string(udata.table, TABLE_ERSATZ_DASH);
+                assert_se(cell = table_get_cell(udata.table, 0, 3));
+                table_set_minimum_width(udata.table, cell, 11);
+                table_set_ersatz_string(udata.table, TABLE_ERSATZ_DASH);
+        } else {
+                r = json_build(&udata.json, JSON_BUILD_EMPTY_OBJECT);
```

Comment:
don't use json_build() if you want to create a trivial object. use `json_variant_new_object(&udata.json, NULL, 0)` instead